main/subunit/APKBUILD

```diff
@@ -1,14 +1,16 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=subunit
 pkgver=1.2.0
 pkgrel=0
 pkgdesc="A streaming protocol for test results"
+options="!check" # Dependencies for checking stuff are in testing/
 url="https://launchpad.net/subunit"
 arch="all"
-license="Apache BSD"
+license="Apache-2.0 BSD-3-Clause"
 depends="python2"
 makedepends="check-dev cppunit-dev perl-dev python2-dev"
-options="!check"
+checkdepends="py-testtools py-fixtures py-hypothesis py-testscenarios bash"
```

Comment:
packages in main cannot depend on packages in testing. We need to either move py-testtools to main or move them all to community. Otherwise good.

Reply:
Not even when they are guarded under !check and only in checkdepends ?

Reply:
yes. unfortunately we don't build our world twice, one without tests and one with. We always run the tests on our builders.